lymph/node.py

```diff
@@ -147,6 +142,7 @@ def __init__(
         allowed_states: Optional[List[int]] = None,
     ) -> None:
         """Create a new lymph node level."""
+        self.allowed_states = allowed_states

```

Comment:
Same as in the `Tumor` class: Not an issue, but... why? 😅

Reply:
There was a bug before that I fixed like this, but it is indeed not needed anymore